open-machine-learning-jupyter-book/assignments/machine-learning-productionization/debugging-in-classification.ipynb

```diff
@@ -68,11 +68,204 @@
     },
```

Comment:
<p>UsageError: Line magic function <code>%hide_result</code> not found.</p><p>In your jupyter environment,this command might not be a standard or built-in command .</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/700/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>